index.md

```diff
@@ -841,12 +842,18 @@ Under DevOps, there be no more "silos". Sometimes, the two teams be mergin' into
 
 DevOps teams use tools to make the journey smoother and quicker, like a good set of sails on a calm day. They use a bunch o' tools called the "DevOps toolchain" to help 'em tackle the important stuff, like continuous integration, continuous delivery, automation, and workin' together like a well-oiled crew!
 
-Even the security teams be gettin' in on the action, with a thing called "DevSecOps". It's like havin' a lookout on the masthead, keepin' a keen eye out for danger at all times!
+In some DevOps models, quality assurance and security teams may also become more tightly integrated with development and operations and throughout the application lifecycle. When security is the focus of everyone on a DevOps team and even the security teams be gettin' in on the action, with a thing called "DevSecOps". It's like havin' a lookout on the masthead, keepin' a keen eye out for danger at all times!
 
 The DevOps journey be like a never-endin' loop, always improvin' and changin', like the wind direction on a voyage. It's made up of eight phases, representin' the processes, capabilities, and tools needed for development and operations. And throughout it all, the teams be workin' and communicatin' together, like a good crew on a ship!
 
 So hoist the sails and set course for DevOps, me hearties! Yarrr!
 
+### **How DevOps Works?**
+
+Under the DevOps model, development and operations teams are no longer siled. Sometimes, these two teams are merged into a single team where the engineers work across the entire application lifecycle, from development and test to deployment to operations, and develop a range of skills not limited to a single function.
```

Comment:
Is this meant to be siloed rather than siled? That is the only issue I can pick up so far.

Reply:
Sorry, it's my mistake. It is siled.

Reply:
Cool just wanted to check so I can get a demo ready and show you how I translate it. Don't want to mess up any of your writing in the process if possible.

Reply:
No problem. If you have any questions. I am happy to answer them.